metrics_proto.go

```diff
@@ -45,12 +45,17 @@ import (
 
 var errNilMetric = errors.New("expecting a non-nil metric")
 var errNilMetricDescriptor = errors.New("expecting a non-nil metric descriptor")
+var percentileLabelKey = &metricspb.LabelKey{
+	Key:         "percentile",
+	Description: "the value at a given percentile of a distribution",
+}
 
 type metricProtoPayload struct {
 	node             *commonpb.Node
 	resource         *resourcepb.Resource
 	metric           *metricspb.Metric
 	additionalLabels map[string]labelValue
+	convertedMetrics []*metricspb.Metric // Summary metrics.
```

Comment:
Is it possible to combine this field and `metric` field above? e.g use one `metrics []*metricspb.Metric` field in `metricProtoPayload`.

Reply:
should be. Let me give it a shot.